imports/ui_3/views/components/comments-section.js

```diff
@@ -71,6 +71,14 @@ Template.Comments_section.events({
 
 const MAX_LENGTH = 400;
 
+Template.Comment.onRendered(function () {
+  Waypoint.refreshAll();
```

Comment:
Nem kene itt valahova egy subscriptionsReady?
Amikor rajzoljuk ki a kommenteket, es rendereljuk ki oket sorba, mindegyik utan az osszes Waypoint-ot ujra fogjuk szamolni.
Sok comment eseten nem fogja ez baromira belassitani a board kirenderelest?

Reply:
Biztos meglehet. Csak hivatkozni kene tudni innen a parent-ra.
A Blaze-ben is fel lehet hivatkozni ugy hogy ../
akkor itt is biztos lehet valahogy.
Akkor ugy jo lenne, ha csak akkor fut le, ha mar subscriptionReady van, es nem minden egyes komment utan.